cypress/support/index.ts

```diff
@@ -0,0 +1 @@
+import "cypress-file-upload";
```

Comment:
Replace for `import "./commands"` as [per the docs](https://www.npmjs.com/package/cypress-file-upload#installation).